### DISCUSSION TITLE
Can not create object user with special "access key" and "secret key"

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#11173] [Q&A]
we can only use "gen access key" and "gen secret key".

**Comment by parth-gr** [comment]
Here is the example https://docs.ceph.com/en/latest/man/8/radosgw-admin/#examples
Hope it answers your question,
If not can you elaborate it more

**Reply by zhucan** [reply]
I want to use the crd of the objectuser not radosgw-admin cmd.

**Reply by parth-gr** [reply]
Object.yaml in the examples doesn't have a configuration for this?

**Reply by zhucan** [reply]
no examples, maybe not support, because no any config in the crd.

**Reply by zhucan** [reply]
It's possible to support it? @parth-gr

**Reply by parth-gr** [reply]
Yaa I see there is no parameter for this to set,
Can you give more insights on what things are been restricted currently to be performed and can be fixed by giving that ability

**Reply by zhucan** [reply]
firstly: we need support to create bucket with special user that it was created by "object-user.yaml".
Before we use rook-ceph, we use minio as backend s3 storage to support the applications, such as "argo" and "gitlab ci cache"; the applicatoin of argo persisent the "ak", "sk" to the postgres; but at the moment, we want to change the backend storage from minio to ceph throught migrate data; But the user want to keep the information of the object user keep the same.

so we need to create the  same user with the same ak, sk  in the storage of minio.

**Reply by parth-gr** [reply]
@BlaineEXE ^^
PLease take a look what we can do
